src/server/debug.rs

```diff
@@ -746,9 +799,152 @@ impl<ER: RaftEngine> DebuggerImpl<ER> {
             None => Err(Error::NotFound(format!("region {}", region_id))),
         }
     }
+
+    fn exec_flashback_region(
+        &self,
+        region_id: u64,
+        version: u64,
+        start_ts: TimeStamp,
+        commit_ts: TimeStamp,
+        f: impl Fn(&Self, u64, Vec<u8>, Vec<u8>, Context, bool, TimeStamp, TimeStamp) -> Result<()>,
+    ) -> Result<()> {
+        let timeout = Duration::from_secs(FLASHBACK_TIMEOUT);
```

Comment:
Is this timeout duration too conservative? Although we have a default configuration where each region size does not exceed 96 MB, in some scenarios with large regions, will the time consumption exceed this value?

Reply:
Okay for keeping consistent with TiDB, I will change that to `1800s`